docs/appendices/release-notes/5.9.0.rst

```diff
@@ -106,5 +106,5 @@ Administration and Operations
 - Added support for :ref:`Shared Access Signatures (SAS) tokens <sql-create-repo-azure-sas-token>`
   as an alternative for authentication for :ref:`Azure repositories <sql-create-repo-azure>`.
 
-- Added ``id`` column to the :ref:`sys.snapshots <sys-snapshots>` table which
-  exposes snapshot UUID.
+- Added ``id``, ``reason``, ``total_shards`` and ``include_global_state``
+  columns to the :ref:`sys.snapshots <sys-snapshots>` table.
```

Comment:
I only kept column names in change entry. I think users can check referenced page to check details, otherwise would have to copy the same info and make the change entry bloated.